### PR TITLE
Added warning about CSI snapshot durability to README

### PR DESCRIPTION
**Pull request by dsu-igeek**
Signed-off-by: Dave Smith-Uchida <dsmithuchida@vmware.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,17 @@ These plugins are currently in beta as of the [Velero 1.4 release][1] and will r
...
+# WARNING
+CSI Snapshots are a standard Kubernetes mechanism for taking snapshots.  The actual implementation of snapshots
+varies by storage vendor.  For disaster recovery, snapshots must be stored in a durable store, such as an S3
+bucket, tape library, etc. and not just on the primary storage.  If the snapshot is only stored on the primary
+storage and the storage is corrupted or destroyed the backup will be lost.
+
+CSI snapshots on AWS EBS, Azure managed disks and Google Cloud Persistent Disk are durable and can be safely
+used for backup.
+
+For all other storage systems, please check with your storage vendor.  Restic backup to object storage
```
Comment (ashish-amarnath) [inline README.md:19]: I think 
>Restic backup to object storage should be used if the storage system does not support durable snapshots.

can be removed as this file doesn't have any context of restic.

Comment (dsu-igeek) [reply README.md:19]: I'd like to give the users a pointer to options they can use instead.  Can you give me some wording that will do that?

Comment (ashish-amarnath) [reply README.md:19]: "If your storage vendor doesn't support durable snapshot storage you may want to consider [Velero's Restic Integration](https://velero.io/docs/v1.5/restic/)"

Comment (dsu-igeek) [reply README.md:19]: Great, that's good, thanks!

**Review by nrb — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -7,6 +7,18 @@ These plugins are currently in beta as of the [Velero 1.4 release][1] and will r
...
+varies by storage vendor.  For disaster recovery, snapshots must be stored in a durable store, such as an S3
+bucket, tape library, etc. and not just on the primary storage.  If the snapshot is only stored on the primary
+storage and the storage is corrupted or destroyed the backup will be lost.
+
+CSI snapshots on AWS EBS, Azure managed disks and Google Cloud Persistent Disk are durable and can be safely
+used for backup.
+
+For all other storage systems, please check with your storage vendor.  If your storage vendor doesn't support 
+durable snapshot storage you may want to consider 
+[Velero's Restic Integration](https://velero.io/docs/v1.5/restic)
```
Comment (ashish-amarnath) [inline README.md:21]: Can you please switch the link to `https://velero.io/docs/restic/`

**Review by ashish-amarnath — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (dsu-igeek) [reply README.md]: Done, thanks for catching that.

**Review by carlisia — APPROVED** [review]
👍

**Review by ashish-amarnath — APPROVED** [review]

**Pull request merged**